src/client.py

```diff
@@ -0,0 +1,303 @@
+import logging
+import numpy as np
+from math import tau
+from enum import Enum
+import rerun as rr
+
+from src.entity import (
+    Entity,
+    MeshFromPath
+)
+from src.scene import Scene
+
+logger = logging.getLogger(__name__)
+
+class Archetype(Enum):
+    ARROWS3D = 0
+    BoxeS3D = 1
+    CAPSULES3D = 2
+    LINESTRIPS3D = 3
+    MESH3D = 4
+    MESH_FROM_PATH = 5
+    POINTS3D = 6
+    
+
+class Client:
+    def __init__(self):
+        self.gui = Gui()
+
+class Gui:
+    def __init__(self, windowName="python-pinocchio", recordingName="world"):
```

Comment:
```suggestion
    def __init__(self, windowName: str ="python-pinocchio", recordingName: str ="world"):
```

Reply:
Does that mean I have to get rid of some asserts if I specify the parameter type ?
```
def createWindow(self, name: str):
    assert isinstance(name, str), "Parameter 'name' must be a string"
```

Reply:
those things are orthogonal: you could have either
- assert checking types at runtime
- type hints, allowing tooling too check type statically
- none
-both